linumpy/reconstruction.py

```diff
@@ -51,6 +51,8 @@ def get_tiles_ids(directory, z: int = None):
         tiles_to_process = f"tile_*"
     tiles = list(input_directory.rglob(tiles_to_process))
     tiles = [t for t in tiles if t.name.startswith('tile_')]
```

Comment:
iterating only once through the tiles might be better?
```suggestion
    tiles = [t for t in tiles if (t.name.startswith('tile_') and not os.path.isfile(t))]
```

Reply:
Definitely